xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -369,15 +374,52 @@ def importfromfile(objtypelist, objlist, location,dryrun=None,version=None,updat
             key, value = env.split('=')
             vardict[key] = value
 
+    if 'OBJNAME' in list(jinjavarlist):
```

Comment:
I would like there is a method to fill-up all built-in variables, then we can have an central place for them.
It would be better to understand the code logic and easy to add/remove them.

Reply:
fixed